script/install-deps-windows.bat

```diff
@@ -1,5 +1,6 @@
 @echo off
-
+set VCPKG_ROOT_DIR=%1
```

Comment:
What happens if not specified on the call to the script? We overwrite variables defined on the system level to empty?

Reply:
yes, later in the script 
`if defined VCPKG_ROOT_DIR if /i not "%VCPKG_ROOT_DIR%"=="" set "VcPkgDir=%VCPKG_ROOT_DIR%"`

if not defined we continue with the default